pages/common/gcloud-auth.md

```diff
@@ -0,0 +1,29 @@
+# gcloud auth
+
+> Grant and revoke authorization to `gcloud` and manage credentials.
+> See also: `gcloud`.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/auth>.
+
+- Authorize Google Cloud access for the `gcloud` CLI with Google Cloud user credentials and set the current account as active:
+
+`gcloud auth login`
+
+- Authorize Google Cloud access similar to `gcloud auth login` but with service account credentials:
+
+`gcloud auth activate-service-account`
+
+- Manage your Application Default Credentials (ADC) for Cloud Client Libraries:
+
+`gcloud auth application-default`
+
+- List all credentialed accounts:
```

Comment:
Can accounts be 'uncredentialed'? Might need rewording here?

Reply:
We used the documentation's description here, will update it to something like this:

```suggestion
- Display a list of Google Cloud accounts currently authenticated on your system:
```